azure-gaming-pr/reference-architectures/unreal-pixel-streaming-deploying.md

```diff
@@ -13,7 +13,7 @@ ms.prod: azure-gaming
 
 ## Overview
 
-This guide will walk through the process of taking the exported Unreal Pixel Streaming package and deploying it on an Azure GPU Virtual Machine, or multiple VMs if using the Matchmaker service to load balance across many streams on multiple GPU VMs. For a detailed overview of Pixel Streaming and architectures in Azure, see our documentation [here](unreal-pixel-streaming-in-azure.md). For deploying Unreal Pixel Streaming at scale (i.e., 2 to 1000&#39;s of streams) see our documentation [here](unreal-pixel-streaming-at-scale.md), which uses Terraform to deploy your Pixel Streaming solution in Azure with auto-scaling.
+This guide will walk through the process of taking the exported Unreal Pixel Streaming package and deploying it on an Azure GPU Virtual Machine, or multiple VMs if using the Matchmaker service to load balance across many streams on multiple GPU VMs. For a detailed overview of Pixel Streaming and architectures in Azure, see our documentation [here](unreal-pixel-streaming-in-azure.md). For deploying Unreal Pixel Streaming at scale (i.e., 2 to 1000&#39;s of streams) with the easy to deploy Azure Marketplace solution, see the documentation [here](https://docs.unrealengine.com/4.27/ProductionPipelines/CloudDeployments/AzurePixelStreaming/), which allows users to deploy Pixel Streaming in Azure right from the Azure Portal. This now supports Unreal Engine 4 and Unreal Engine 5 apps. See Epic Games' Azure [Marketplace Offer](https://azuremarketplace.microsoft.com/marketplace/apps/epicgames.unreal-pixel-streaming?tab=Overview).
```

Comment:
Consider breaking your sentence up so it reads better.

"To learn how to deploy Unreal Pixel Streaming at scale, from two to thousands of streams, with the easy to deploy Azure Marketplace solution, see [Deploying Unreal Engine 4 Pixel Streaming on Azure](https://docs.unrealengine.com/4.27/ProductionPipelines/CloudDeployments/AzurePixelStreaming/). This marketplace solution allows users to deploy Pixel Streaming in Azure, right from the Azure Portal. It also supports both Unreal Engine 4 and 5 apps. For more information, see [Unreal Pixel Streaming in Azure marketplace](https://azuremarketplace.microsoft.com/marketplace/apps/epicgames.unreal-pixel-streaming?tab=Overview)."

Reply:
Thank you. I'm removing the 4 out of the title of the docs, as this supports 5 now as well. Epic will be updating their Azure docs as well.